File: app/src/main/java/com/soumya/wwdablu/zomatobuddy/model/search/UserRating.java
package com.soumya.wwdablu.zomatobuddy.model.search;

public class UserRating{
	private String aggregateRating;
	private String ratingColor;
	private String ratingText;
	private String votes;

	public void setAggregateRating(String aggregateRating){
		this.aggregateRating = aggregateRating;
	}

	public String getAggregateRating(){
		return aggregateRating;
	}

	public void setRatingColor(String ratingColor){
		this.ratingColor = ratingColor;
	}

	public String getRatingColor(){
		return ratingColor;
	}

	public void setRatingText(String ratingText){
		this.ratingText = ratingText;
	}

	public String getRatingText(){
		return ratingText;
	}

	public void setVotes(String votes){
		this.votes = votes;
	}

	public String getVotes(){
		return votes;
	}
}

File: app/src/main/java/com/soumya/wwdablu/hungry/model/network/search/UserRating.kt
package com.soumya.wwdablu.hungry.model.network.search

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class UserRating (

        @SerializedName("aggregate_rating")
        var aggregateRating: String,

        @SerializedName("rating_text")
        var ratingText: String,

        @SerializedName("rating_color")
        var ratingColor: String,

        @SerializedName("rating_obj")
        var ratingObj: RatingObj,

        var votes: Int

) : Parcelable
